hack/generate_gcp_kpt_data.py

```diff
@@ -0,0 +1,61 @@
+"""Regenerate tests."""
+
+import fire
+import logging
+import os
+import shutil
+import subprocess
+
+class Generator:
+  @staticmethod
+  def write_gcp_kpt(kpt="kpt"):
+    """Create test data based on running the kpt commands.
+
+    This will allow us to see any diffs if we refactor the commands
+    """
+    repo_root = subprocess.check_output(["git", "rev-parse", "--show-toplevel"])
+    repo_root = repo_root.decode()
+    repo_root = repo_root.strip()
+
+    data_dir = os.path.join(repo_root, "tests", "testdata_gcp_kpt")
+    if os.path.exists(data_dir):
+      shutil.rmtree(data_dir)
+
+    logging.info("Removing directory %s", data_dir)
+
+    subdirs = ["v2", "Kptfile"]
```

Comment:
We also call kpt setters on GCP stack, shall we include it in the snapshot?

Reply:
Lets do it in a follow on PR just so we can get this merged.
My initial PR isn't converting the setters in stacks.